mucgpt-frontend/src/components/AssistantDialogs/AssistantEditorPage/AssistantEditorPage.module.css

```
@@ -1,98 +1,198 @@
 }
 
-.settingsGrid {
-    display: grid;
-    grid-template-columns: repeat(2, minmax(0, 1fr));
-    gap: 24px;
+.actionStatus::before {
+    content: "";
+    width: 6px;
+    height: 6px;
+    border-radius: var(--borderRadiusCircular);
+    background-color: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)